.pipelines/swiftv2-long-running/scripts/create_vnets.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+set -e
+trap 'echo "[ERROR] Failed while creating VNets or subnets. Check Azure CLI logs above." >&2' ERR
+
+SUBSCRIPTION_ID=$1
+LOCATION=$2
+RG=$3
+
+az account set --subscription "$SUBSCRIPTION_ID"
+
+# VNets and subnets
+VNET_A1="cx_vnet_a1"
+VNET_A2="cx_vnet_a2"
+VNET_A3="cx_vnet_a3"
+VNET_B1="cx_vnet_b1"
+
+A1_S1="10.10.1.0/24"
+A1_S2="10.10.2.0/24"
+A1_PE="10.10.100.0/24"
+
+A2_MAIN="10.11.1.0/24"
+
+A3_MAIN="10.12.1.0/24"
+
+B1_MAIN="10.20.1.0/24"
+
+# A1
+az network vnet create -g "$RG" -n "$VNET_A1" --address-prefix 10.10.0.0/16 --subnet-name s1 --subnet-prefix "$A1_S1" -l "$LOCATION" --output none \
+ && echo "Created $VNET_A1 with subnet s1"
+az network vnet subnet create -g "$RG" --vnet-name "$VNET_A1" -n s2 --address-prefix "$A1_S2" --output none \
+ && echo "Created $VNET_A1 with subnet s2"
+az network vnet subnet create -g "$RG" --vnet-name "$VNET_A1" -n pe --address-prefix "$A1_PE" --output none \
+ && echo "Created $VNET_A1 with subnet pe"
+
+# A2
```

Comment:
Also I just realized, what is the purpose of vnet 2,3 and b1?
I see A1 has a storage account attached to it via a private endpoint to A1's subnet, but what are A2,3 and B1 for? (I see we peer them all together, but are there supposed to be any resources on A2,3,B1?